OpenRA.Game/FileFormats/VocLoader.cs

```diff
@@ -0,0 +1,348 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OpenRA.FileFormats
+{
+	public class VocLoader : Stream
```

Comment:
Definitely keep this on a separate branch for the future, but for now it would be better to merge a simple parser that isn't overloaded with a bunch of functionality that it doesn't yet need.